tests/mpi_54rank/test_external_grid_2x2.py

```diff
@@ -0,0 +1,155 @@
+import math
```

Comment:
Is there a way to combine the 1x1 and 2x2 test into a single python test accepting an argument - or is that not the convention we are adopting in Pace?